lua/tardis/controls/sh_control_float.lua

```diff
@@ -8,7 +8,7 @@ TARDIS:AddControl({
         end
     end,
     serveronly=true,
-    power_independent = false,
+    power_independent = true,
```

Comment:
I haven't found any reasons for it to fail in the code
Will test though

The point of the change is to allow the player to **disable** anti-gravs when powered-down

So if you mean the `CanTurnOnFloat` hook, that's okay since there's no `CanTurnOffFloat` hook

Reply:
**UPD**: double-checked, you can disable anti-gravs when power is disabled
that was the goal
not turning on but turning off (so that you don't get stuck)

Reply:
Ah okay gotcha, didn't realise that was the reason I just tried turning it on